docs/number/parseHumanDuration.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: parseHumanDuration
+description: Parses a human duration string into milliseconds
+---
```

Comment:
Add the version 

```suggestion
---
title: parseHumanDuration
description: Parses a human duration string into milliseconds
since: 12.3.0
---
```